cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrapSettings.scala

```diff
@@ -19,6 +19,9 @@ final class ClusterBootstrapSettings(config: Config) {
       config.getString(key) != s"<$key>"
   }
 
+  val managementBasePath: Option[String] =
+    Option(config.getString("akka.management.http.base-path")).filter(_.trim.nonEmpty)
```

Comment:
A bit of a duplication of the management settings class, but this seemed least disruptive. Open to other ideas.